From 0dcc5689389a1c0d832b5b281c2ba8f72c589846 Mon Sep 17 00:00:00 2001
From: czietz <czietz@gmx.net>
Date: Sun, 10 Feb 2019 15:32:58 +0000
Subject: [PATCH 1/4] Eliminate binding for form_error() from EmuDesk

After commit 5fb3e63 form_error() is not called anymore. Eliminating the binding saves some bytes of ROM space.
---
 desk/aesbind.h | 1 -
 desk/gembind.c | 8 --------
 2 files changed, 9 deletions(-)

diff --git a/desk/aesbind.h b/desk/aesbind.h
index 7629ced1..d764ec28 100644
--- a/desk/aesbind.h
+++ b/desk/aesbind.h
@@ -87,7 +87,6 @@ WORD form_do(OBJECT *form, WORD start);
 WORD form_dial(WORD dtype, WORD ix, WORD iy, WORD iw, WORD ih,
                WORD x, WORD y, WORD w, WORD h);
 WORD form_alert(WORD defbut, const BYTE *astring);
-WORD form_error(WORD errnum);
 WORD form_center(OBJECT *tree, WORD *pcx, WORD *pcy, WORD *pcw, WORD *pch);
 /*
 WORD form_keybd(OBJECT *form, WORD obj, WORD nxt_obj, WORD thechar,
diff --git a/desk/gembind.c b/desk/gembind.c
index b891c287..2bb8c360 100644
--- a/desk/gembind.c
+++ b/desk/gembind.c
@@ -498,14 +498,6 @@ WORD form_alert(WORD defbut, const BYTE *astring)
     return gem_if(AES_CTRL_CODE(FORM_ALERT, 1, 1, 1));
 }
 
-
-WORD form_error(WORD errnum)
-{
-    FM_ERRNUM = errnum;
-    return gem_if(AES_CTRL_CODE(FORM_ERROR, 1, 1, 0));
-}
-
-
 WORD form_center(OBJECT *tree, WORD *pcx, WORD *pcy, WORD *pcw, WORD *pch)
 {
     FM_FORM = (LONG)tree;
-- 
2.17.0.windows.1

